pkg/scheduler/core/generic_scheduler.go

```diff
@@ -701,11 +736,15 @@ func (g *genericScheduler) prioritizeNodes(
 		}
 	}
 
-	if klog.V(10) {
-		for i := range result {
+	for i := range result {
+		span.AddEvent(ctx, "Node Score",
+			core.KeyValue{Key: "Node", Value: core.String(result[i].Name)},
```

Comment:
in a log message below it is called `Host`. Curious why use different name here?

Reply:
No reason, just a typo on my part. Host and node are used pretty interchangeably throughout the scheduler code